pkg/controller/issuer/googlecasclusterissuer_controller.go

```diff
@@ -42,9 +42,7 @@ type GoogleCASClusterIssuerReconciler struct {
 
 // +kubebuilder:rbac:groups=cas-issuer.jetstack.io,resources=googlecasclusterissuers,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Maybe remove the RW permissions here, unless the controller does need to update spec or meta?